build-deps.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eu
+
+# pip install numpy==1.25.1 cython==0.29.36 # required for MOAB install (silly)
+
+# It's important to use LLVM clang rather than Apple clang
+# CC=/opt/homebrew/Cellar/llvm/16.0.6/bin/clang
+# CXX=/opt/homebrew/Cellar/llvm/16.0.6/bin/clang++
+
+NUM_CORES=2
+
+# 5.5.0 breaks with pyne std::isnan bug
+MOAB_TAG='5.3.0'
```

Comment:
5.5.1 has been released, might fix the pyne std::isnan bug
```suggestion
MOAB_TAG='5.5.1
```